instances/latest/terminologies/technique/superResolutionMicroscopy.jsonld

```diff
@@ -0,0 +1,16 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/technique/superResolutionMicroscopy",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/Technique",
+  "definition": null,
```

Comment:
Try to adapt something form wikipedia entry: https://en.wikipedia.org/wiki/Super-resolution_microscopy

Reply:
done/deleted instance as already part of other PR